services/portal/home/homeSequences.js

```diff
@@ -9,15 +11,24 @@ const I = actor();
  */
 module.exports = {
   async login(userAcct = user.mainAcct) {
+    const systemUseConfigured = homeQuestions.systemUsePopupConfigured();
+    homeTasks.goToHomepage();
```

Comment:
nice, we could remove all the `goToHomepage()` calls before `login()` calls in other tests too. but it doesn't have to be in this PR